### PR TITLE
Basic Steganography implementation 

### PR DESCRIPTION
**Pull request by BreadGenie**
Basic Steganography implementation with minimum styling

What's implemented 
- [x]  Encoding the secret message to the image
- [x]  Decoding the secret message from the image

### CODE REVIEW

File: src/steganography.js [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+  reader.onload = (event) => {
+    let dataUrl = event.target.result;
+    let img = new Image();
+    img.onload = () => {
+      let ctx = document.getElementById('canvas').getContext('2d');
+      ctx.canvas.width = img.width;
+      ctx.canvas.height = img.height;
+      ctx.drawImage(img, 0, 0);
+    }
+    img.src = dataUrl;
```
Comment (anandbaburajan) [inline src/steganography.js]: You could maybe check if a non-image file is loaded. Feel free to open an issue and someone else might be interested to work on this.

**Comment by anandbaburajan** [comment]
@BreadGenie I just tried the app, works good! Looks like the project is almost over already, haha.

The code and structure looks good too. You can merge the PR already if you want. Or do you want to write tests too?
I think you can open an issue for adding tests and someone else can go through your code and write tests after understanding the working.

There are a few other members too, right? Do they plan to work on improving the design or something?

Also, you setup the vercel bot integration with the project right? I'm curious bc I'm not familiar with it.

Great work by the way! Thanks for contributing!

File: src/steganography.js (hunk 1 above)
Comment (BreadGenie) [reply src/steganography.js]: Thanks for the suggestion 
I'll look into it and merge it after
Thanks for the review!

**Comment by BreadGenie** [comment]
> @BreadGenie I just tried the app, works good! Looks like the project is almost over already, haha.
> 
> The code and structure looks good too. You can merge the PR already if you want. Or do you want to write tests too?
> I think you can open an issue for adding tests and someone else can go through your code and write tests after understanding the working.
> 
> There are a few other members too, right? Do they plan to work on improving the design or something?
> 
> Also, you setup the vercel bot integration with the project right? I'm curious bc I'm not familiar with it.
> 
> Great work by the way! Thanks for contributing!

I'm planning to add tests at a later time. Most probably after the design team have worked on improving the design

I did vercel since I'm familiar with it than other alternatives

You're welcome :3

File: src/steganography.js (hunk 1 above)
Comment (anandbaburajan) [reply src/steganography.js]: That's great! But it's okay and much better if you open an issue and let someone else work on this. They'd get an opportunity to understand the code and you can guide them if they've doubts.

**Pull request merged**